src/data/sample.json

```diff
@@ -0,0 +1,417 @@
+      "approxDuration": "3 months",
+      "formedFromInterestGroup": ["1","6"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "2",
+      "name": "Apollo",
+      "description": "Project Apollo was a comprehensive space program led by NASA and the United States government, aimed at achieving the goal of landing a man on the moon and returning him safely to Earth before the end of the 1960s",
+      "approxDuration": "5 months",
+      "formedFromInterestGroup": ["2","5"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "3",
+      "name": "Rocket to Mars",
+      "description": "We are going to Mars! is the fourth planet from the Sun and is the second-smallest planet in the Solar System, being larger than only Mercury. In the English language, Mars is named for the Roman god of war.",
+      "approxDuration": "10 years",
+      "formedFromInterestGroup": ["3","4"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    }
+  ],
+  "users": [
+    {
+      "id": "1",
+      "username": "John Doe",
+      "email": "john@doe.com",
+      "appRole": "admin",
+      "projectsOwned": ["1"],
+      "isMemberOfProjects": ["1"],
+      "targetIndustryRoles": ["CTO", "Lead Engineer"],
+      "isMemberOfInterestGroups" :["1","2"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "2",
+      "username": "Jane Smith",
+      "email": "jane@smith.com",
+      "appRole": "user",
+      "projectsOwned": [],
+      "isMemberOfProjects": ["1", "2"],
+      "targetIndustryRoles": ["2"],
+      "isMemberOfInterestGroups" :["3","4"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "3",
+      "name": "Bob Johnson",
+      "email": "bob@johnson.com",
+      "appRole": "user",
+      "projectsOwned": ["2", "3"],
+      "isMemberOfProjects": ["2", "3"],
+      "targetIndustryRoles": ["3", "4"],
+      "isMemberOfInterestGroups" :["1","5"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "4",
+      "name": "Alice Williams",
+      "email": "alice@williams.com",
+      "appRole": "user",
+      "projectsOwned": [],
+      "isMemberOfProjects": ["3"],
+      "targetIndustryRoles": ["10", "11"],
+      "isMemberOfInterestGroups" :["5","6"],
+      "dateAdded": "2023-06-15T08:45:00Z",
+      "dateModified": "2023-06-18T10:20:00Z"
+    },
+    {
+      "id": "5",
+      "name": "Charlie Brown",
+      "email": "charlie@brown.com",
+      "appRole": "user",
+      "projectsOwned": [],
+      "isMemberOfProjects": ["1"],
+      "targetIndustryRoles": ["12"],
+      "isMemberOfInterestGroups" :["4"],
+      "dateAdded": "2023-06-15T08:45:00Z",
```

Comment:
can you please:
- change `dateAdded` to `createdAt`
- change `dateModified` to `updatedAt`
Those are more standard nomenclature for timestamps.
Thanks!